### PR TITLE
Update URL of instructions + re-enable support of 'branch' HTML get param

### PR DESCRIPTION
**Pull request by jmirabel**
This is required for #66 to work.

### CODE REVIEW

File: download.html [hunk 1]
```diff
@@ -35,8 +36,19 @@
...
+
+function getParameterByName(name) {
+    name = name.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]");
+    var regex = new RegExp("[\\?&]" + name + "=([^&#]*)"),
+        results = regex.exec(location.search);
+    return results === null ? "" : decodeURIComponent(results[1].replace(/\+/g, " "));
+}
+
+var branch = getParameterByName('branch');
+document.getElementById('installation_branch').val(branch);
```
Comment (jmirabel) [inline download.html:51]: @nim65s some time ago, you remove those lines. I am not sure why you did it and could find a note explaining it. Was it on purpose ?

**Comment by jmirabel** [comment]
The result can be visualized at http://jmirabel.github.io/hpp-doc/download.html

File: download.html (hunk 1 above)
Comment (nim65s) [reply download.html]: Either I found that everything was working for me without that, and I removed it on purpose, or that was a mistake I don't know how to explain

Comment (jmirabel) [reply download.html]: This code makes it possible to send a link with the correct version:
http://jmirabel.github.io/hpp-doc/download.html?branch=master

**Pull request merged**